crayfish/plugin.py

```diff
@@ -45,6 +45,8 @@
 from .illuvis import upload_dialog
 from .styles import style_with_black_lines, classified_style_from_colormap, classified_style_from_interval
 
+import webbrowser
```

Comment:
Should be at the top of the list. see https://www.python.org/dev/peps/pep-0008/#imports